tools/importers/openScienceDataImporter.php

```diff
@@ -124,7 +124,7 @@
 }
 
 // Write the report to file.
-$path = $config->getSetting('base') . 'project/data_import/';
+$path = '/var/www/loris/project/data_import/';
```

Comment:
Note: because we are using v20.1, there is not $config variable avaibale in the generic_includes so I had to hardcode the path for PREVENT-AD.